diff --git a/wordpress/wp-admin/includes/plugin.php b/wordpress/wp-admin/includes/plugin.php
--- a/wordpress/wp-admin/includes/plugin.php
+++ b/wordpress/wp-admin/includes/plugin.php
@@ -2111,8 +2111,17 @@ function get_plugin_page_hookname( $plugin_page, $parent_page ) {
  */
 function user_can_access_admin_page() {
 	global $pagenow, $menu, $submenu, $_wp_menu_nopriv, $_wp_submenu_nopriv,
+		$typenow, $_post_type_pages,
 		$plugin_page, $_registered_pages;
 
+	// Early check for post_type pages
+	if (
+		isset( $_post_type_pages[ "$pagenow?post_type=$typenow" ] ) &&
+		current_user_can( $_post_type_pages[ "$pagenow?post_type=$typenow" ] )
+	) {
+		return true;
+	}
+
 	$parent = get_admin_page_parent();
 
 	if ( ! isset( $plugin_page ) && isset( $_wp_submenu_nopriv[ $parent ][ $pagenow ] ) ) {
diff --git a/wordpress/wp-admin/menu.php b/wordpress/wp-admin/menu.php
--- a/wordpress/wp-admin/menu.php
+++ b/wordpress/wp-admin/menu.php
@@ -115,13 +115,12 @@ $types   = (array) get_post_types(
 		'show_in_menu' => true,
 	)
 );
+
+$_post_type_pages = array();
+
 $builtin = array( 'post', 'page' );
 foreach ( array_merge( $builtin, $types ) as $ptype ) {
-	$ptype_obj = get_post_type_object( $ptype );
-	// Check if it should be a submenu.
-	if ( true !== $ptype_obj->show_in_menu ) {
-		continue;
-	}
+	$ptype_obj           = get_post_type_object( $ptype );
 	$ptype_menu_position = is_int( $ptype_obj->menu_position ) ? $ptype_obj->menu_position : ++$_wp_last_object_menu; // If we're to use $_wp_last_object_menu, increment it first.
 	$ptype_for_id        = sanitize_html_class( $ptype );
 
@@ -150,6 +149,14 @@ foreach ( array_merge( $builtin, $types ) as $ptype ) {
 		$edit_tags_file = "edit-tags.php?taxonomy=%s&amp;post_type=$ptype";
 	}
 
+	$_post_type_pages[ $ptype_file ]    = $ptype_obj->cap->edit_posts;
+	$_post_type_pages[ $post_new_file ] = $ptype_obj->cap->create_posts;
+
+	// Check if it should be a submenu.
+	if ( true !== $ptype_obj->show_in_menu ) {
+		continue;
+	}
+
 	if ( in_array( $ptype, $builtin, true ) ) {
 		$ptype_menu_id = 'menu-' . $ptype_for_id . 's';
 	} else {
