diff --git a/wordpress/wp-includes/load.php b/wordpress/wp-includes/load.php
--- a/wordpress/wp-includes/load.php
+++ b/wordpress/wp-includes/load.php
@@ -574,7 +574,11 @@ function wp_debug_mode() {
 	}
 
 	if ( WP_DEBUG ) {
-		error_reporting( E_ALL );
+		if ( defined( 'WP_DEBUG_ERROR_LEVEL' ) ) {
+			error_reporting( WP_DEBUG_ERROR_LEVEL );
+		} else {
+			error_reporting( E_ALL );
+		}
 
 		if ( WP_DEBUG_DISPLAY ) {
 			ini_set( 'display_errors', 1 );
