diff --git a/wordpress/wp-content/plugins/sg-security/sg-security.php b/wordpress/wp-content/plugins/sg-security/sg-security.php
index 5ce97d8..aa2568c 100644
--- a/wordpress/wp-content/plugins/sg-security/sg-security.php
+++ b/wordpress/wp-content/plugins/sg-security/sg-security.php
@@ -7,7 +7,7 @@
  * @link              http://www.siteground.com/
  *
  * @wordpress-plugin
- * Plugin Name:       Security Optimizer - The All-In-One WordPress Protection Plugin
+ * Plugin Name:       Security Optimizer
  * Plugin URI:        https://siteground.com
  * Description:       Security Optimizer by SiteGround is the all-in-one security solution for your WordPress website. With the carefully selected and easy to configure functions the plugin provides everything you need to secure your website and prevent a number of threats such as brute-force attacks, compromised login, data leaks and more.
  * Version:           1.4.6
